actors/builtin/shared.go

```diff
@@ -43,9 +43,10 @@ type MinerAddrs struct {
 	Worker addr.Address
 }
 
-// This type is required to construct a miner (in power_actor) and change a miners multiaddress (in miner_actor)
-// the type is defined here to work around a circular dependency between the actors.
+// These types are required to construct a miner (in power_actor) and change a miners multiaddress/peerId (in miner_actor)
+// these types are defined here to work around a circular dependency between the actors.
 type Multiaddrs = []byte
+type PeerID = []byte
```

Comment:
Let's just move them to `abi`. You then don't need to talk about the dependency, just state what each one represents.